Subject: PS3: Make ps3av.h usable from user space

From: Masashi Kimoto <Masashi_Kimoto@hq.scei.sony.co.jp>

The user applications to manage the PS3 AV modes can use values
defined in this header.

CC: Masashi Kimoto <Masashi_Kimoto@hq.scei.sony.co.jp>
Signed-off-by: Geert Uytterhoeven <Geert.Uytterhoeven@sonycom.com>
Signed-off-by: Geoff Levand <geoffrey.levand@am.sony.com>
---
 include/asm-powerpc/Kbuild  |    1 +
 include/asm-powerpc/ps3av.h |    2 ++
 2 files changed, 3 insertions(+)

--- a/include/asm-powerpc/Kbuild	2007-10-31 17:43:31.000000000 -0400
+++ b/include/asm-powerpc/Kbuild	2007-10-31 17:43:59.000000000 -0400
@@ -34,6 +34,7 @@ unifdef-y += elf.h
 unifdef-y += nvram.h
 unifdef-y += param.h
 unifdef-y += posix_types.h
+unifdef-y += ps3av.h
 unifdef-y += ptrace.h
 unifdef-y += seccomp.h
 unifdef-y += signal.h
--- a/include/asm-powerpc/ps3av.h	2007-10-31 17:43:31.000000000 -0400
+++ b/include/asm-powerpc/ps3av.h	2007-10-31 17:43:59.000000000 -0400
@@ -324,6 +324,7 @@
 #define PS3AV_MODE_RGB				0x0020
 
 
+#ifdef __KERNEL__
 /** command packet structure **/
 struct ps3av_send_hdr {
 	u16 version;
@@ -723,4 +724,5 @@ extern void ps3av_register_flip_ctl(void
 				    void *flip_data);
 extern void ps3av_flip_ctl(int on);
 
+#endif /* __KERNEL__ */
 #endif	/* _ASM_POWERPC_PS3AV_H_ */
